support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListData.tsx

```diff
@@ -0,0 +1,81 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { neutral } from '@guardian/source-foundations';
+import { SvgCrossRound, SvgTickRound } from '@guardian/source-react-components';
+
+const greyedOut = css`
+	color: ${neutral[60]};
+
+	svg {
+		fill: ${neutral[60]};
+	}
+`;
+
+const boldText = css`
+	font-weight: bold;
+`;
+
+type TierUnlocks = {
+	lowerTier: boolean;
+	higherTier: boolean;
+};
+
+export type CheckListData = {
+	icon: JSX.Element;
+	text?: JSX.Element;
+	maybeGreyedOut: null | SerializedStyles;
+};
+
+export const getSvgIcon = (isUnlocked: boolean): JSX.Element =>
+	isUnlocked ? (
+		<SvgTickRound isAnnouncedByScreenReader size="small" />
+	) : (
+		<SvgCrossRound isAnnouncedByScreenReader size="small" />
+	);
+
+export const checkListData = ({
+	lowerTier,
+	higherTier,
+}: TierUnlocks): CheckListData[] => {
+	return [
+		{
+			icon: getSvgIcon(lowerTier),
+			text: (
+				<p>
+					<span css={boldText}>Uninterrupted reading. </span> No more yellow
+					banners
+				</p>
+			),
+			maybeGreyedOut: lowerTier ? null : greyedOut,
```

Comment:
You could define 2 variables at the top of this `checkListData` function :

```
const maybeGreyedOutLowerTier = lowerTier ? null : greyedOut`;
const maybeGreyedOutHigherTier = higherTier ? null : greyedOut`;
```

Then you could use the variables values, it'd save making the same calculations multiple times in this function.